code/game/machinery/poolcontroller.dm

```diff
@@ -248,17 +251,15 @@
 		if(panel_open)
 			dat += wires()
 
-
 		dat += text({"
-					<BR><TT><B>Pool Controller Panel</B></TT><BR><BR>
-					<B>Current temperature :</B> [temperature]<BR>
-					</B>Drain is [drainable ? "active" : "unactive"]</B> <BR>"})
-
-//herpderp			if(beaker)
-//herpderp				dat += "<a href='?src=\ref[src];beaker=1'>Remove Beaker</a><br>"
-//herpderp				dat += "<B><span class='good'>Duplicator filled with [cur_reagent].</span></B><BR><BR><BR>"
-//herpderp			if(!beaker)
-//herpderp				dat += "<B><span class='bad'>No beaker loaded</span></B><BR><BR><BR>"
+			<BR><TT><B>Pool Controller Panel</B></TT><BR><BR>
+			<B>Current temperature :</B> [temperature]<BR>
+			<BR><B>Drain is [drainable ? "active" : "unactive"]</B> <BR>"})
+		if(beaker)
+			dat += "<a href='?src=\ref[src];beaker=1'>Remove Beaker</a><br>"
+			dat += "<B><span class='good'>Duplicator filled with [cur_reagent].</span></B><BR><BR><BR>"
```

Comment:
you may want to not use that cur reagent var perhaps,just directly access the reagent inside the beaker. Keep in mind that ids usually don't have spaces and it might be ugly to see "filled with space_lube", use the reagent name instead.